### PR TITLE
Rewrite the README to reflect the v6 changes

### PR DESCRIPTION
**Pull request by rdwebdesign**
**Note:** This PR depends on the graphics repo PR to include the new images.

---

### What does this PR aim to accomplish?

Change the README file to reflect the v6 changes, showing the new features and removing the deprecated ones.

### How does this PR accomplish the above?

Adding new images a and changing texts where needed.

---
**By submitting this pull request, I confirm the following:**

1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against.
2. I have commented my proposed changes within the code and I have tested my changes.
3. I am willing to help maintain this change if there are issues with it later.
4. It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
5. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
6. I have checked that another pull request for this purpose does not exist.
7. I have considered, and confirmed that this submission will be valuable to others.
8. I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
9. I give this submission freely, and claim no ownership to its content.

---
- [x] I have read the above and my PR is ready for review. *Check this box to confirm*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
+    <a href="https://discourse.pi-hole.net/">Discourse Forum</a> |
+    <a href="https://pi-hole.net/donate">Donate</a>
+  </div>
+  <br>
+  <br>
+</div>
+
+# Pi-hole Web Interface
+
+<img src="https://pi-hole.github.io/graphics/Screenshots/v6/v6_dashboard.png" alt="Pi-hole Web interface">
```
Comment (yubiuser) [inline README.md:21]: Depending on https://github.com/pi-hole/web/pull/3139 we might need another new image

**Review by yubiuser — CHANGES_REQUESTED** [review]
What is the browser-stack-badge for? I don't think we use that service...

__

I would remove the link to disourse on how to access the web interface.

__

We can now also access via `https`

____

Remove "forgot your password?"

___

Passwort protection - maybe add 2FA + Screenshot

File: README.md (hunk 1 above)
Comment (rdwebdesign) [reply README.md:21]: I already prepared a branch with new images, just in case #3139 is approved.

**Comment by rdwebdesign** [comment]
> What is the browser-stack-badge for?

I really don't know. 
This badge was there since a long time ago. I will remove it because we don't use this service.

**Comment by rdwebdesign** [comment]
> What is the browser-stack-badge for? I don't think we use that service...
- Done: badge removed

> I would remove the link to disourse on how to access the web interface.
- Done: Agree... link removed

> We can now also access via https
- Done: note added to make the use of http/https clear;

> Remove "forgot your password?"
- Done: text removed

> Passwort protection - maybe add 2FA + Screenshot
- I will change the "login screen" image in a future PR.

**Review by yubiuser — DISMISSED** [review]

**Comment by PromoFaux** [comment]
> This badge was there since a long time ago. I will remove it because we don't use this service.

We might not have used it recently, but we have a sponsored plan with the service, so perhaps put it back for the time being (I think it was a condition of the plan)

**Comment by rdwebdesign** [comment]
I will wait until we decide what will happen with https://github.com/pi-hole/web/pull/3139.

**Comment by yubiuser** [comment]
> sponsored plan with the service

Do you want to re-enable the service to check the web interface?

**Review by DL6ER — APPROVED** [review]

**Pull request merged**